### DISCUSSION TITLE
What about roles?

### DISCUSSION THREAD

**Discussion by Wuerike** [redpanda-data/connect#2065] [General]
It took me hours¹ to noticed the gcp pubsub output verifies if the topic exists and that my pipeline wasnt working because I set just the `pubsub.publisher` role instead of also use `pubsub.viewer` .

Is it a bad idea improve the docs whit this level o info?
.
.
.
.
¹ It took me longer than it should because the latest version is omitting the error reason, so I just noticed was a role issue when I downgraded to 4.15.0. But it's fine, I've already seen in some issue this problem will be resolved in the next release

**Comment by Jeffail** [comment]
Hey @Wuerike yeah I think that'd be a good idea. Ideally with all of the cloud based components, including all the AWS ones, we'd list exactly which permissions each component requires for normal operation.
